modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp

```diff
@@ -37,9 +37,11 @@ SearchContainer<Object> searchContainer = itemSelectorViewDescriptorRendererDisp
 		var="entriesSearch"
 	>
 		<liferay-ui:search-container-row
+			ariaLabel="select-the-item-and-close-the-modal"
```

Comment:
@pat270 Actually since this is a row that we are adding a keyboard action to, it's probably better if we make the label "Press enter to select the item and close the modal.".